18_links_lab/linkslab.txt

```diff
@@ -7,14 +7,17 @@ Preparation: Perform the following  (no need to write the commands you used):
 1. Perform the following and write the commands you used, from within links directory:
 
   - Create a symbolic link named file3 that points to "file1"
-  Answer: 
+  Answer: ln -s $(realpath file1) file3
 
   - Create a hard link named file4 that points to "file2"
-  Answer: 
+  Answer: ln file2 file4
 
   - Create a new directory named "tmp" in your home folder:
   Answer:
+  mkdir ../tmp
 
 2. Move "file3" to the new "~/tmp" directory you just created. Can you still use it to see the original "file1" ? Why not? How would you create a symbolic link that would survive the transfer to ~/tmp ?
+I've created symbolic link with absolute path, therefore I can see the original "file1"
```

Comment:
that's correct! :) but note that relative paths on symlinks are also very useful in case you are moving the entire parent directory.